Tests/LicensePlistTests/Entity/FileReader/SwiftPackageFileReaderTests.swift

```diff
@@ -111,12 +112,12 @@ class SwiftPackageFileReaderTests: XCTestCase {
               "kind" : "remoteSourceControl",
               "location" : "https://github.com/jpsim/Yams.git",
               "state" : {
-                "revision" : "f47ba4838c30dbd59998a4e4c87ab620ff959e8a",
-                "version" : "5.0.5"
+                "revision" : "d41ba4e7164c0838c6d48351f7575f7f762151fe",
+                "version" : "6.1.0"
               }
             }
           ],
-          "version" : 2
+          "version" : 3
```

Comment:
In the new Xcode, the Package.resolved version was update to v3, so I included the change in this update.